test/linux.uts

```diff
@@ -8,55 +8,6 @@
 
 + Linux only test
 
-= TCP client automaton
-~ automaton netaccess linux needs_root
-* This test retries on failure because it often fails
-
-from __future__ import print_function
-import os
-import time
-import signal
-
-from scapy.modules.six.moves import range
-
-def handler(signum, stack_frame):
-    raise Exception("Timer expired !")
-
-tmp = signal.signal(signal.SIGALRM, handler)
-
-SECDEV_IP4 = "203.178.141.194"
-IPTABLE_RULE = "iptables -%c INPUT -s %s -p tcp --sport 80 -j DROP"
-
-# Drop packets from SECDEV_IP4
-assert(os.system(IPTABLE_RULE % ('A', SECDEV_IP4)) == 0)
```

Comment:
@guedou Hi, could you explain what those do ?
The tests pass on my Linux machine, but fail on Travis :/ If I read this correctly the rule discarded HTTP packets ?!

Reply:
It drops packets received from the IP address 203.178.141.194 to the TCP port 80. This firewall rule prevents the kernel from receiving the TCP segment and answering with a RST as the port is closed.

Reply:
Any idea why the test would be failing ?

Reply:
Is this test disable on current master? I did not see any issue with this test.

Reply:
I found out what the issue was: `SECDEV_IPV4` is the ip of `www.kame.org`.